backend/app/models/log_records.py

```diff
@@ -13,7 +14,7 @@ class LogRecords(db.Model):
     flagged = db.Column(db.Boolean, nullable=False)
     attn_to = db.Column(db.Integer, db.ForeignKey("users.id"), nullable=True)
     # TODO: replace open String fields with VarChar(NUM_CHARS)
-    note = db.Column(db.String, nullable=True)
+    note = db.Column(db.String, nullable=False)
```

Comment:
some backend changes need to be made with this, because there are some clauses that check if a note is there. this doesn't have to be done this PR but just something to note